apiserver/facades/client/application/application.go

```diff
@@ -1188,6 +1189,16 @@ You'll need to deploy a new charm rather than upgrading if you need this change.
 Juju on containers does not support updating node selectors (configured from charm devices).
 The new charm's metadata contains updated device declarations.
 You'll need to deploy a new charm rather than upgrading if you need this change.
+`[1:]
+
+	upgradedApplicationHasUnitsMessage = `
```

Comment:
We shouldn't be using user messages server side. We should be using a typed error and rendering the message client side. We need a new apiserver error code for application should not have units